configs/pluginsConfig.json

```
@@ -555,32 +598,45 @@
       "name": "MapTemplates",
       "title": "Map Templates",
       "dependencies": [
-          "BurgerMenu"
+        "SidebarMenu"
       ]
-    }, {
+    },
+    {
       "name": "UserExtensions",
       "glyph": "1-user-add",
       "title": "plugins.UserExtensions.title",
       "hidden": true,
       "description": "plugins.UserExtensions.description",
-      "dependencies": ["BurgerMenu"]
-    }, {
+      "dependencies": [
+        "SidebarMenu"
+      ]
+    },
+    {
       "name": "UserSession",
       "glyph": "floppy-save",
       "title": "plugins.UserSession.title",
       "description": "plugins.UserSession.description",
-      "dependencies": ["BurgerMenu"]
-    }, {
+      "dependencies": [
+        "SidebarMenu"
+      ]
+    },
+    {
       "name": "StreetView",
       "glyph": "road",
       "title": "plugins.StreetView.title",
       "description": "plugins.StreetView.description",
-      "dependencies": ["Toolbar"],
+      "dependencies": [
+        "Toolbar"
```

I think it needs to be modified this way.
```suggestion
        "SidebarMenu"
```